RxCocoa/iOS/UIPickerView+Rx.swift

```diff
@@ -40,6 +61,20 @@
                 }
             return ControlEvent(events: source)
         }
+        
+        public func items<O: ObservableType,
+                          Adapter: RxPickerViewDataSourceType & UIPickerViewDataSource & UIPickerViewDelegate>(adapter: Adapter)
+            -> (_ source: O)
+            -> Disposable where O.E == Adapter.Element {
+                return { source in
+                    let delegateSubscription = self.setDelegate(adapter)
```

Comment:
Maybe we should rename `subscribeProxyDataSource` to something else. It has some nice mutation checks. Take a look at "Proxy changed from the time it was first set.\nOriginal...".

I've remembered all of the small checks we are doing in that method but forgot about the main functionality of that method which is subscribing 😂.

I think we can live for now without those small checks, but it would be nice to figure out how to refactor them in future.